### DISCUSSION TITLE
Works perfectly on Comfast CF-758F

### DISCUSSION THREAD

**Discussion by WDutra22** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#259] [General]
It worked perfectly on the Comfast CF-758F.
Realtek RTL8811CU Chip
MacOS Ventura 13.0
Macbook Pro Mid 2012

**Comment by chris1111** [comment]
I add to the support list thanks @WDutra22

**Comment by m-krol-github** [comment]
Hi, it's an old topic but I'll try to ask you how you made it to work.
For me it's displaying only "no wifi USB..."   and nothing but this, I'm trying different USBs but no any luck, on windows its at first time showing Cd Drive with software and drivers and after installing it's became Network Device in Devices... Bummer dunno what to do now....
